cli/man/TEMPLATE.1.md.example

```diff
@@ -0,0 +1,69 @@
+% COMMAND(1) Cargill, Incorporated | Splinter Commands
+
+<!--
+This file should be named the full command (with dashes between subcommands)
+with an extension of `1.md`. For example `splinter-cert-generate.1.md` is the
+markdown file that will generated into the man page for
+`splinter generate cert`.
+-->
+NAME
+====
+
+**command-name** — about from command help
+
+SYNOPSIS
+========
+| **command** \[**FLAGS**\] \[**OPTIONS**\]
+
+DESCRIPTION
+===========
+Paragraph or two that describes what the command does, explains how it fits in
+the Splinter environment, provides big-picture information on who uses it and
+why (or when), and lists prerequisites or other considerations.
+
+FLAGS
+=====
+<!--
+List of flags and options (in alphabetical order) that describes what it does,
+what arguments it takes, and the default value/behavior. If appropriate, explain
+how/when it's used, with tips, gotchas, and very short examples.
+-->
```

Comment:
To clarify the expected info in this section, this comment should be for the flags only (they don't have arguments, for example). My suggestion:

```
List of flags (in alphabetical order) that describes what each flag does 
and the default value/behavior. If appropriate, explain
how/when it's used, with tips, gotchas, and very short examples.
```

Reply:
updated